### ISSUE TITLE
surf_quickie missing textures on linux

### ISSUE THREAD

**Issue by granis48** [momentum-mod/game#1826]
### Describe the bug

This is an issue in csgo as well: https://github.com/ValveSoftware/csgo-osx-linux/issues/2874

tldr; 
When a map has packed custom materials in a folder named something thats just one letter/character long the game will look for any referenced textures in the root materials folder rather than the subfolder they're in. 

I made this test map to showcase the problem together with Isark:
[materialfoldertest.zip](https://github.com/momentum-mod/game/files/8882229/materialfoldertest.zip) (compiled in csgo hammer)  
Later @BIRD#7588 verified this problem persists in momentum mod as well (as of posting this issue)
 

### How To Reproduce

Load up momentum mod in linux, 
Launch surf_quickie or the linked test map, 
note the missing/different textures. 

### Issue Map

[surf_quickie_fix.zip](https://github.com/momentum-mod/game/files/8882271/surf_quickie_fix.zip) - surf map from css
[materialfoldertest.zip](https://github.com/momentum-mod/game/files/8882229/materialfoldertest.zip) - test map made in csgo

### Expected Behavior

In the test map you will see what the materials are supposed to look like.  
The expected behavior is for linux to parse file paths correctly inside packed vmts

### Operating System

Linux

### Renderer

DX11 (default)

**Comment by Zylviij** [comment]
![image](https://user-images.githubusercontent.com/7347274/173152909-b0a6f031-ffcf-4fad-b8b6-4c2445d6fc13.png)

**Comment by lvaness** [comment]
Tested it on Steam, is fixed now
